jobs/devex/jenkins-bump-version/Jenkinsfile

```diff
@@ -0,0 +1,127 @@
+                    choices: ['rhaos-3.7-rhel-7', 'rhaos-3.6-rhel-7'].join('\n'),
+                    defaultValue: 'rhaos-3.7-rhel-7'
+                ],
+                [
+                    name: 'MAIL_LIST_SUCCESS',
+                    description: 'Success Mailing List',
+                    $class: 'hudson.model.StringParameterDefinition',
+                    defaultValue: 'jupierce@redhat.com,smunilla@redhat.com,ahaile@redhat.com,gmontero@redhat.com,bparees@redhat.com,pep@redhat.com'
+                ],
+                [
+                    name: 'MAIL_LIST_FAILURE',
+                    description: 'Failure Mailing List',
+                    $class: 'hudson.model.StringParameterDefinition',
+                    defaultValue: 'jupierce@redhat.com,smunilla@redhat.com,ahaile@redhat.com,gmontero@redhat.com,bparees@redhat.com,pep@redhat.com'
+                ],
+                [
+                    name: 'TARGET_NODE',
+                    description: 'Jenkins agent node',
+                    $class: 'hudson.model.StringParameterDefinition',
+                    defaultValue: 'openshift-build-1'
+                ],
+                [
+                    $class: 'hudson.model.BooleanParameterDefinition', 
+                    defaultValue: false,
+                    description: 'Mock run to pickup new Jenkins parameters?', 
+                    name: 'MOCK'
+                ],            
+            ]
+        ],
+        disableConcurrentBuilds()
+    ]
+)
+
+def mail_success() {
+
+    jenkins_major = JENKINS_VERSION.tokenize('.')[0].toString()
+
+    distgit_link = "http://pkgs.devel.redhat.com/cgit/rpms/jenkins-${jenkins_major}-plugins/?h=rhaos-${OCP_RELEASE}-rhel-7"
+
+    mail(
+        to: "${MAIL_LIST_SUCCESS}",
+        from: "aos-cd@redhat.com",
+        replyTo: 'jupierce@redhat.com',
+        subject: "jenkins plugins RPM for OCP ${OCP_RELEASE} updated in dist-git",
+        body: """The Jenkins plugins RPM for OCP ${OCP_RELEASE} has been updated in dist-git:
+${distgit_link}
+
+Minimum Jenkins version: ${JENKINS_VERSION}
+
+Plugin list:
+${PLUGIN_LIST}
+
+Plugin RPM update job: ${env.BUILD_URL}
+""");
+}
+
+def mail_failure(err) {
+
+    mail(
+        to: "${MAIL_LIST_FAILURE}",
+        from: "aos-cd@redhat.com",
+        replyTo: 'jupierce@redhat.com',
+        subject: "Error during jenkins plugin RPM update on dist-git",
+        body: """The job to update the jenkins plugins RPM in dist-git encountered an error:
+${err}
+
+Jenkins job: ${env.BUILD_URL}
+""");
+}
+
+node(TARGET_NODE) {
+
+    checkout scm
+
+    def buildlib = load( "pipeline-scripts/buildlib.groovy" )
+    buildlib.kinit()       // Sets up credentials for dist-git access
+
+    try {
+        stage ("prepare workspace") {
+
+            // Temporary file to write the plugin list to
+            tmpdir = pwd tmp:true
+            scripts_dir = "${env.WORKSPACE}/hacks/bump-jenkins"
```

Comment:
Looks like actual directory is `bump-jenkins-version`

Reply:
✔️